diff --git a/node_modules/@shelf/jest-mongodb/setup.js b/node_modules/@shelf/jest-mongodb/setup.js
index e2e29cd..83da48f 100644
--- a/node_modules/@shelf/jest-mongodb/setup.js
+++ b/node_modules/@shelf/jest-mongodb/setup.js
@@ -26,7 +26,7 @@ module.exports = async () => {
 
   // Set reference to mongod in order to close the server during teardown.
   global.__MONGOD__ = mongod;
-  process.env.MONGO_URL = mongoConfig.mongoUri;
+  process.env.MONGODB_URL = mongoConfig.mongoUri;
 };
 
 function getMongodbMemoryOptions() {
